app/helpers/markdown_helper.rb

```diff
@@ -0,0 +1,12 @@
+module MarkdownHelper
+  def markdown(text)
+    markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML,
+                                       no_intra_emphasis: true,
+                                       fenced_code_blocks: true,
+                                       autolink: true,
+                                       tables: true,
+                                       underline: true,
+                                       highlight: true)
+    markdown.render(text).html_safe
```

Comment:
Here it would be very good to have a test case checking for XSS.

XSS, if you don't know, basically means: can the user inject javascript into the page?